### PR TITLE
[Ethereum] [Set protocol] Add cETH and cWBTC to component price feed

### PR DESCRIPTION
**Pull request by jackiep00**
Added two extra steps to the insertion process. cETH and cWBTC aren't often traded, so their prices from dex data are either inaccurate or missing. To get their prices we pull directly from the Compound exchange rate events.

After making this change, we should truncate the table and re-run the backfill again.

I've checked that:

* [X] the query produces the intended results
* [X] the folder name matches the schema name
* [X] the schema name exists in Dune
* [X] views are prefixed with `view_`, functions with `fn_`.
* [X] the filename matches the defined view, table or function and ends with .sql
* [X] each file has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`

### CODE REVIEW

**Comment by jeff-dude** [comment]
working through backfill with up-to-date function now 👍

File: ethereum/setprotocol_v2/insert_daily_component_prices.sql [hunk 1]
```diff
@@ -240,6 +240,214 @@ with initial_components as (
   
 )
 SELECT count(*) + r INTO r from rows;
+-------------------------------------------------------------------------------------------------------------------------
+-- Step 3: Update prices for cETH using Compound-specific query
```
Comment (jeff-dude) [inline ethereum/setprotocol_v2/insert_daily_component_prices.sql:244]: it appears step 3 has duplicates on the values you implied in the 'ON CONFLICT' statement during inserts. please refer to this query to identify duplicates:
https://dune.com/queries/699027

i'm unable to load due to this failure

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: ethereum/setprotocol_v2/insert_daily_component_prices.sql (hunk 1 above)
Comment (jackiep00) [reply ethereum/setprotocol_v2/insert_daily_component_prices.sql:244]: Alright fixed, sorry about that, I was keeping two copies of the code, one using `dune_user_generated` and the other using `setprotocol_v2` and i didn't propagate all the changes over

**Review by jeff-dude — APPROVED** [review]
@jackiep00 backfill complete & cron job enabled again 🙌

**Pull request merged**